tests/network-bridge-firewall

```diff
@@ -18,6 +18,17 @@ lxc network create lxdbr0 \
     ipv4.address=192.0.2.1/24 \
     ipv6.address=2001:db8::1/64 \
     ipv4.dhcp.ranges=192.0.2.2-192.0.2.199
+
+echo "=> Ensure that IPv6 CIDR size checks in LXD are correct"
+# Check that we produce a warning if user sets a prefix larger than /64
+lxc network set lxdbr0 ipv6.address=2001:db8::1/48
+lxc warning list | grep "IPv6 networks with a prefix larger than 64"
```

Comment:
A `lxc warning delete --all` is probably needed here.

Reply:
Nice catch!
But actually, as far as I understand we do cleanup old warnings automatically when configuration issue is resolved:
https://github.com/canonical/lxd/blob/6c31513b084098f3891ee9a9312c1d6f8a1380e8/lxd/network/driver_bridge.go#L1518

Reply:
Oh, I didn't know about this, thanks!